<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="glWindow">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>9</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>459</width>
       <height>389</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="xy_group">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>231</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>X Y position</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>95</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="xy_verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="x_horizontalLayout">
          <item>
           <widget class="QLabel" name="x_position_label">
            <property name="text">
             <string>x position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="x_positon_lineedit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="y_horizontalLayout">
          <item>
           <widget class="QLabel" name="y_position_label">
            <property name="text">
             <string>y position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="y_positon_lineedit"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="start_button">
        <property name="text">
         <string>Start</string>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
